core/commands/add.go

```diff
@@ -30,11 +30,9 @@ const (
 
 var AddCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline: "Add a file to ipfs.",
+		Tagline: "Add a file or directory hierarchy to ipfs.",
```

Comment:
I'd drop hierarchy from this, `Add a file or directory to ipfs` is short sweet and to the point.